It does not build on rhel 6

```
cc  -o c_tests/unit 'c_tests/1a332ed@@unit@exe/unit.c.o' -L/builddir/DCV_JHBUILD/inst/lib -Wl,--as-needed -Wl,--no-undefined -fstack-protector -g -O2 -fno-strict-aliasing -Wformat -D_FORTIFY_SOURCE=2 -Wl,-z,relro -Wl,-z,now -pthread -Wl,--start-group /builddir/DCV_JHBUILD/sources/dcv-color-primitives-0.1.3/target/debug/libdcv_color_primitives.a -ldl -Wl,--end-group
/builddir/DCV_JHBUILD/sources/dcv-color-primitives-0.1.3/target/debug/libdcv_color_primitives.a(std-9d59e25e2eb2384d.std.5u3styxu-cgu.0.rcgu.o): In function `now':
/rustc/5e1a799842ba6ed4a57e91f7ab9435947482f7d8//src/libstd/sys/unix/time.rs:387: undefined reference to `clock_gettime'
/builddir/DCV_JHBUILD/sources/dcv-color-primitives-0.1.3/target/debug/libdcv_color_primitives.a(std-9d59e25e2eb2384d.std.5u3styxu-cgu.0.rcgu.o): In function `std::sync::mpsc::blocking::WaitToken::wait_max_until::h29132948868d6846':
/rustc/5e1a799842ba6ed4a57e91f7ab9435947482f7d8/src/liballoc/sync.rs:(.text._ZN3std4sync4mpsc8blocking9WaitToken14wait_max_until17h29132948868d6846E+0x45): undefined reference to `clock_gettime'
/builddir/DCV_JHBUILD/sources/dcv-color-primitives-0.1.3/target/debug/libdcv_color_primitives.a(std-9d59e25e2eb2384d.std.5u3styxu-cgu.0.rcgu.o): In function `now':
/rustc/5e1a799842ba6ed4a57e91f7ab9435947482f7d8//src/libstd/sys/unix/time.rs:387: undefined reference to `clock_gettime'
/rustc/5e1a799842ba6ed4a57e91f7ab9435947482f7d8//src/libstd/sys/unix/time.rs:387: undefined reference to `clock_gettime'
/rustc/5e1a799842ba6ed4a57e91f7ab9435947482f7d8//src/libstd/sys/unix/time.rs:387: undefined reference to `clock_gettime'
/builddir/DCV_JHBUILD/sources/dcv-color-primitives-0.1.3/target/debug/libdcv_color_primitives.a(std-9d59e25e2eb2384d.std.5u3styxu-cgu.0.rcgu.o):/rustc/5e1a799842ba6ed4a57e91f7ab9435947482f7d8//src/libstd/sys/unix/time.rs:387: more undefined references to `clock_gettime' follow
collect2: ld returned 1 exit status
ninja: build stopped: subcommand failed.
```

--- a/c_tests/meson.build
+++ b/c_tests/meson.build
@@ -4,6 +4,7 @@ dcp_deps = [
   libdl_dep,
   ws2_32_dep,
   userenv_dep,
+  librt_dep,
 ]
 
 unit = executable(

--- a/meson.build
+++ b/meson.build
@@ -108,6 +108,19 @@ libdl_dep = cc.find_library('dl', required: is_win == false)
 ws2_32_dep = cc.find_library('ws2_32', required: is_win)
 userenv_dep = cc.find_library('userenv', required: is_win)
 
+clock_gettime_test_code = '''
+  #include <time.h>
+  struct timespec t;
+  int main (int argc, char ** argv) {
+    return clock_gettime(CLOCK_REALTIME, &t);
+  }'''
+librt_dep = []
+if cc.links(clock_gettime_test_code, name : 'clock_gettime')
+  # nothing to link to
+elif cc.links(clock_gettime_test_code, args : '-lrt', name : 'clock_gettime in librt')
+  librt_dep = cc.find_library('rt')
+endif
+
 subdir('include')
 subdir('c_tests')
 

